module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md

```diff
@@ -105,16 +90,53 @@ Name           : Application.Read.All
 Description    : Allows the app to read all applications and service principals without a signed-in user.
 ```
 
-## Parameters
+### Example 3. Search for permissions with exact match
+
+```powershell
+Find-EntraPermission -SearchString 'User.Read.All' -ExactMatch
+```
 
+```Output
+   PermissionType: Delegated
+
+Id                                   Consent Name          Description
+--                                   ------- ----          -----------
+a154be20-db9c-4678-8ab7-66f6cc099a59 Admin   User.Read.All Allows the app to read the full set of profile properties, reports, and ma…
```

Comment:
Use fake GUIDs in this md file.

Reply:
@KenitoInc: Permission strings and Role Template IDs are exempted as they are public info.

![image](https://github.com/microsoftgraph/entra-powershell/assets/132555836/4c52ec70-bfc1-4a97-8878-0450773f191b)